pages/linux/pacstrap.md

```diff
@@ -0,0 +1,36 @@
+# pacstrap
+
+> Install packages using Pacman to the specified new root directory.
+> More information: <https://man.archlinux.org/man/pacstrap.8>.
+
+- Install the `base` package, Linux kernel and firmware for common hardware:
+
+`pacstrap {{path/to/new/root}} {{base}} {{linux}} {{linux-firmware}}`
+
+- Install packages without copy the host’s mirrorlist to the target:
+
+`pacstrap -M {{path/to/new/root}} {{packages}}`
+
+- Use an alternate configuration file for Pacman:
+
+`pacstrap -C {{path/to/pacman.conf}} {{path/to/new/root}} {{packages}}`
+
+- Install packages using the package cache on the host, rather than the target:
```

Comment:
At first I was confused by this statement.  I think this is clearer:
```suggestion
- Install packages using the package cache on the host instead of on the target:
```